task2.py

```diff
@@ -0,0 +1,47 @@
+import json
+from pprint import pprint
+
+
+# function to get textures of given set
+def getTextures(_set,source):
+  textureName = str(_set["name"])+'_'+str(_set["id"])
```

Comment:
Use space between the operators and variables. 

Check if _set["id"] and _set["name"] is not None because str(None) is an error.